### PR TITLE
[workloads] Shorten the workload name even more

### PR DESCRIPTION
**Pull request by lewing**

### CODE REVIEW

File: src/workloads/workloads.csproj [hunk 1]
```diff
@@ -79,8 +79,8 @@
     <!-- Shorten package names to avoid long path issues in Visual Studio -->
     <ItemGroup>
       <!-- These need to be in order - matching rules will try in order. -->
-      <ShortNames Include="Microsoft.NET.Workload.Mono.ToolChain.Manifest">
-        <Replacement>MonoToolChainManifest</Replacement>
+      <ShortNames Include="Microsoft.NET.Workload.Mono.ToolChain.">
```
Comment (filipnavara) [inline src/workloads/workloads.csproj:82]: Too much shortening on this line? :-)

**Review by hoyosjs — DISMISSED** [review]
Sigh... This limit is painful

File: src/workloads/workloads.csproj (hunk 1 above)
Comment (joeloff) [reply src/workloads/workloads.csproj:82]: So we'll end up with mono.manifest or something like that?

Comment (joeloff) [reply src/workloads/workloads.csproj:82]: BTW, I noticed there's already a VSIX package in VS with an ID called "Mono", so as long as we have another qualifier we should be okay.

Comment (lewing) [reply src/workloads/workloads.csproj:82]: should end up as MonoManifest-7.0.100

**Review by radical — APPROVED** [review]

**Pull request merged**